src/sot-dynamic-pinocchio.cpp

```diff
@@ -215,6 +215,12 @@ DynamicPinocchio( const std::string & name)
 				"string (signal name)","string (joint name)");
     addCommand("createVelocity",
 	       makeCommandVoid2(*this,&DynamicPinocchio::cmd_createVelocitySignal,docstring));
+ 
+    docstring = docCommandVoid2("Create an acceleration (matrix homo) signal only for one joint.",
```

Comment:
Acceleration and velocity are not homogeneous matrices. They are vectors. Could you please correct the docs as well?

Reply:
Yeah, sorry, I copied the info from the function `createAccelerationSignal` but did not check if it was right…
I corrected both.